tests/cupy_tests/statics_tests/test_meanvar.py

```diff
@@ -179,3 +179,84 @@ def test_std_axis_ddof(self, xp, dtype):
     def test_external_std_axis_ddof(self, xp, dtype):
         a = testing.shaped_arange((2, 3, 4), xp, dtype)
         return xp.std(a, axis=1, ddof=1)
+
+
+@testing.parameterize(
+    *testing.product({
+        'shape': [(3, 4), (4, 3, 5)],
+        'axis': [None, 0, 1],
+        'keepdims': [True, False],
+        'ddof': [0, 1]
+    }))
+@testing.gpu
+class TestNanVarStd(unittest.TestCase):
+
+    @testing.for_all_dtypes(no_float16=True, no_complex=True)
```

Comment:
Could you add float16 test?

Reply:
To avoid numerical errors, `float16` tests must be simple and small. `rtol = 1e-4` can be used. Right?

Reply:
Tests added